src/render.js

```diff
@@ -13,8 +13,13 @@ import options from './options';
  */
 export function render(vnode, parentDom, replaceNode) {
 	if (options._root) options._root(vnode, parentDom);
-	let oldVNode = replaceNode && replaceNode._children || parentDom._children;
 
+	let oldVNode = replaceNode && replaceNode._children || parentDom._children;
+	let isHydrating = false;
+	if (replaceNode === true) {
```

Comment:
Random though (not sure if it will work): We could try using `null` for `replaceVNode` so that it stays in the same type for v8. That way we wouldn't need to reset it to `undefined` two lines later.

Reply:
Good idea, implemented!